### ISSUE TITLE
[Bug] Should not generate typeparam for non-generic classes

### ISSUE THREAD

**Issue by lonix1** [kasecato/vscode-docomment#76]
The `typeparam` here should not be generated, because the class is not generic:

````csharp
/// <summary>
/// 
/// </summary>
/// <typeparam name="string"></typeparam>    <--- this line should not be generated
public class Foo : Bar<string> { }
````

Also, when generating XML documentation during building, this will result in warnings:
> Foo.cs: warning CS1711: XML comment has a typeparam tag for 'string', but there is no type parameter by that name.

**Comment by kasecato** [comment]
@lonix1 I fixed it and released `v0.1.8`. Please check it. Thanks

**Comment by lonix1** [comment]
@kasecato Thanks! :smile:
